dlk/python/dlk/templates/src/func/conv2d.cpp

```diff
@@ -102,7 +99,7 @@ void conv1x1_kn2row(T input[],
 
    assert(p.input_height > 0);
    assert(p.input_width > 0);
-
```

Comment:
Please remove this kind of white space changes.

Reply:
I will fix it.

Reply:
fixed